### ISSUE TITLE
me command error

### ISSUE THREAD

**Issue by chenjing1294** [memcached/memcached#1100]
```
set key3 1 1000 2
ab
STORED
me key3
ME key3 exp=-992 la=7 cas=3 fetch=no cls=1 size=69
```

Why is exp a negative number? Shouldn't it be a positive number?

**Comment by dormando** [comment]
```
        ret = snprintf(resp->wbuf + total, WRITE_BUFFER_SIZE - (it->nkey + 12),
                "exp=%d la=%llu cas=%llu fetch=%s cls=%u size=%lu\r\n",
                (it->exptime == 0) ? -1 : (current_time - it->exptime),
```

looks like it's always been backwards and nobody noticed. I'm betting the people who used it never set TTL's so they didn't notice.

that command was for by-hand debugging and is largely irrelevant since the real meta protocol came in.

leave the issue open and I'll fix it when I get a chance (or anyone reading this can open a PR; maybe add a test)

**Comment by dormando** [comment]
released in 1.6.24
